package.json

```diff
@@ -144,18 +144,20 @@
   },
   "devDependencies": {
     "@types/node": "^13.5.0",
-    "@types/semver": "^6.2.0",
     "coc.nvim": "0.0.74",
+    "npm-run-all": "^4.1.5",
```

Comment:
without this it would look for `npm-run-all` globally (and failing on my computer since I don't have it installed)

Reply:
Ahh, good catch. 👍